.taskcluster.yml

```diff
@@ -225,21 +226,20 @@ tasks:
                         treeherder:
                           symbol: nightly-D
                       metadata:
-                        name: Fenix Nightly Decision Task
+                        name: ${project_name} Nightly Decision Task
                         description: Decision task scheduled by cron task [${cron.task_id}](https://tools.taskcluster.net/tasks/${cron.task_id})
               - $if: 'cron.name == "raptor"'
                 then:
                   $mergeDeep:
                     - {$eval: 'default_task_definition'}
                     - scopes:
                         - $if: 'trust_level == 3'
-                          then: assume:hook-id:project-mobile/fenix-raptor
-                          else: assume:hook-id:project-mobile/fenix-raptor-staging
+                          then: assume:hook-id:project-mobile/${project_name}-raptor
+                          else: assume:hook-id:project-mobile/${project_name}-raptor-staging
                       routes:
                         $if: 'trust_level == 3'
                         then:
                           - notify.email.fenix-eng-notifications@mozilla.com.on-failed
-                          - notify.email.perf-sheriffs@mozilla.com.on-failed
```

Comment:
Can we also send this to perftest-alerts@mozilla.com?

Reply:
Okay!